packages/@expo/cli/src/start/interface/interactiveActions.ts

```diff
@@ -20,10 +20,13 @@ export class DevServerManagerActions {
     // If native dev server is running, print its URL.
     if (this.devServerManager.getNativeDevServerPort()) {
       try {
-        const url = this.devServerManager.getDefaultDevServer().getNativeRuntimeUrl()!;
+        const qrCodeUrl = this.devServerManager.getDefaultDevServer().getQRCodeUrl()!;
```

Comment:
This URL getter function shouldn't be concerned with how 'interactiveActions' chooses to display it, we named it `getNativeRuntimeUrl` because it refers to what it opens.